mediawiki123-apb/roles/provision-mediawiki123-apb/tasks/main.yml

```diff
@@ -55,16 +55,6 @@
         value: postgres
       - name: MEDIAWIKI_SITE_SERVER
         value: '{{ route.route.spec.host }}'
-      - name: POSTGRESQL_HOST
-        value: '{{ postgresql_host }}'
-      - name: POSTGRESQL_DATABASE
-        value: '{{ postgresql_database }}'
-      - name: POSTGRESQL_PORT
-        value: '{{ postgresql_port }}'
-      - name: POSTGRESQL_USER
-        value: '{{ postgresql_user }}'
-      - name: POSTGRESQL_PASSWORD
-        value: '{{ postgresql_password }}'
       image: docker.io/jmontleon/mediawiki123:latest
```

Comment:
Actually as I work on the dockerfile for downstream I realize it's a moot point in this instance. I'll end up pointing it at the official image once it exists.